File: day06.cpp
#include <iostream>
using namespace std;

void printPatter(int n)
{
    for (int i = n; i > 0; i--)
    {
        for (int j = n; j > 0; j--)
        {
            for (int k = i; k > 0; k--)
            {
                cout << j << " ";
            }
        }
        cout << endl;
    }
}

int main()
{
    int n;
    cout << "Enter the number : ";
    cin >> n;

    printPatter(n);
    return 0;
}

File: day06.java
import java.util.Scanner;

public class day06 {

    public static void main(String[] args) {
        int n;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number : ");
        n = scanner.nextInt();

        printPattern(n);
        scanner.close();
    }

    public static void printPattern(int n) {
        for (int i = n; i > 0; i--) {
            for (int j = n; j > 0; j--) {
                for (int k = i; k > 0; k--) {
                    System.out.print(j + " ");
                }
            }
            System.out.println();
        }
    }

}